TSPL.docc/ReferenceManual/Attributes.md

```diff
@@ -1304,6 +1304,53 @@ can increase your binary size and adversely affect performance.
   because of the larger symbol table slowing dyld down.
 -->
 
+### preconcurrency
+
+Apply this attribute to a declaration,
+to suppress strict concurrency checking.
+
+On an import declaration,
+this attribute reduces the strictness of concurrency checking
+for code that uses types from the imported module.
+Specifically,
+types from the imported module
+that aren't explicitly marked as either sendable or nonsendable
+can be used in a context that requires sendable types.
+
+On other declarations,
+this attribute reduces the strictness of concurrency checking
+for code that uses the symbol being declared.
+When you use this symbol in a scope that has minimal concurrency checking,
+you can use members of this type
+that aren't explicitly marked as either sendable or nonsendable
+in a context that requires sendable types.
```

Comment:
It's not really about members of this type. I think of it as not enforcing any of the concurrency-related constraints of the symbol (such as Sendable or a global actor) on uses of type. Let me take a stab at it.

```suggestion
When you use this symbol in a scope that has minimal concurrency checking,
concurrency-related constraints specified by that symbol (such as `Sendable` requirements or global actors) are not checked.
```